src/cosmos/upgrade/v1beta1/query.ts

```diff
@@ -1,89 +1,88 @@
 /* eslint-disable */
-import Long from "long";
-import _m0 from "protobufjs/minimal";
-import { ModuleVersion, Plan } from "./upgrade";
-
+import { Plan, ModuleVersion } from "./upgrade";
+import * as _m0 from "protobufjs/minimal";
+import { DeepPartial, Exact, isSet, Long, bytesFromBase64, base64FromBytes, Rpc } from "../../../helpers";
 export const protobufPackage = "cosmos.upgrade.v1beta1";
-
 /**
  * QueryCurrentPlanRequest is the request type for the Query/CurrentPlan RPC
  * method.
  */
-export interface QueryCurrentPlanRequest {}
 
+export interface QueryCurrentPlanRequest {}
 /**
  * QueryCurrentPlanResponse is the response type for the Query/CurrentPlan RPC
  * method.
  */
+
 export interface QueryCurrentPlanResponse {
   /** plan is the current upgrade plan. */
-  plan?: Plan;
+  plan: Plan;
 }
-
 /**
  * QueryCurrentPlanRequest is the request type for the Query/AppliedPlan RPC
  * method.
  */
+
 export interface QueryAppliedPlanRequest {
   /** name is the name of the applied plan to query for. */
   name: string;
 }
-
 /**
  * QueryAppliedPlanResponse is the response type for the Query/AppliedPlan RPC
  * method.
  */
+
 export interface QueryAppliedPlanResponse {
   /** height is the block height at which the plan was applied. */
   height: Long;
 }
-
 /**
  * QueryUpgradedConsensusStateRequest is the request type for the Query/UpgradedConsensusState
  * RPC method.
- *
- * @deprecated
  */
+
+/** @deprecated */
+
 export interface QueryUpgradedConsensusStateRequest {
   /**
    * last height of the current chain must be sent in request
    * as this is the height under which next consensus state is stored
    */
   lastHeight: Long;
 }
-
 /**
  * QueryUpgradedConsensusStateResponse is the response type for the Query/UpgradedConsensusState
  * RPC method.
- *
- * @deprecated
  */
+
+/** @deprecated */
```

Comment:
I forget why, but deprecated statements are moved into their own comment now. Can look into this.